.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main, test-af-2.10]
```

```suggestion
    branches: [main]
```
Remove branch before merging